counting/etcd.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright (c) 2024 Diagrid Inc.
+Licensed under the MIT License.
+*/
+
+package counting
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
We don't need this package.

```suggestion
```

Reply:
We do. It gives a compilation error now.

Reply:
shouldn't this just be the standard lib: `errors`?